### PR TITLE
Calibration changes and updated FEB delay measurements for Side CRT timing studies

### PR DESCRIPTION
**Pull request by tboone0042**
I've updated a number of things in the CRT calibration code, mostly adding files that allow us to run the calibration on data that comes from the CRT's general decoder as well as helpful macros for executing the full CRT calibration analysis. There are some additions to CrtCal.cc and CrtCal.h that allow the FEB statistics to be included, allowing us to know how many events were present when fitting the ADC spectra to obtain the calibration. Finally, after Francesco Poppi's measurements of the CRT timing signal delay to the Side CRT racks, I recalculated the delay for each FEB based on the cable lengths from the output of the timing fanouts and using a measured conversion factor for the delay per length in the cables. This should hopefully fix some of the issues we were having with timestamps not matching correctly within the CRT.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
 
-project(icaruscode VERSION 09.42.04 LANGUAGES CXX)
+project(icaruscode VERSION 09.43.00 LANGUAGES CXX)
```
Comment (rennney) [inline CMakeLists.txt:18]: I am not sure if this is not reviewed/approved by others by tomorrow these versions will have to be updated to be merged next week. Version will change tomorrow (Friday)with weekly release

File: ups/product_deps [hunk 2]
```diff
@@ -37,8 +37,8 @@ table_fragment_end
 # Add the dependent product and version
 
 product                   version
-sbncode                   v09_42_04
-icarusalg                 v09_42_00
+sbncode                   v09_43_00
+icarusalg                 v09_43_00
```
Comment (rennney) [inline ups/product_deps:41]: I am not sure if this is not reviewed/approved by others by tomorrow these versions will have to be updated to be merged next week. Version will change tomorrow (Friday)with weekly release

**Review by rennney — APPROVED** [review]
I am not an expert in CRT, so I approved the PR; however, if it is not merged tomorrow code versions will have to be updated, otherwise it will get conflict

**Comment by tboone0042** [comment]
This unfortunately lacked a lot of things I wanted to update, I apologize for the bad request, I'm a little new at using github like this

**Comment by rennney** [comment]
I actually just realized that this is not a PR to develop, but develop to your feature branch. I also do not see any changes to the files you've claimed in the description (CrtCal.cc and CrtCal.h) @tboone0042

**Review by rennney — CHANGES_REQUESTED** [review]
based on my last comment It might require some adjustment

**Comment by tboone0042** [comment]
Please ignore this pull request, I have a second one that I submitted that should have most of my file changes. I'm currently working through the comments Bishu gave me there and will try to explain everything there.

**Pull request closed without merging**